test/models/item_test.rb

```diff
@@ -49,11 +49,11 @@ def setup
     assert_not @item.valid?
   end
   
-  test "url should be present" do 
+ # test "url should be present" do 
```

Comment:
Why is this test commented out?

Reply:
I thought that this was the test that we commented out since it wasn't working, and we weren't doing URL stuff at the time.

Reply:
If it's not relevant anymore we should remove it, or if it should be working we should figure out why it isn't.

Reply:
It turns out we were missing a validation for url!